docs/UserGuide.md

```diff
@@ -117,70 +112,98 @@ Examples of usage:
 *Figure 3: Table showing list of possible tags and the Pre-defined constants*
 
 As described in Figure 3, these are the possible tags that can be used with Tracey. E.g `n/`, `cs/` etc.
-In addition, the list of Pre-defined constants are also provided for `Faculty` and `Covid Status`. 
+In addition, the list of Pre-defined constants are also provided for `Faculty` and `Covid Status`.
 
 💡`Tips`:
 * You are able to add optional tags with no restrictions using the `t/` prefix.
 
+### Editing an existing contact
+Edit a contact at a specific index.
+
+Format: `edit INDEX n/NAME ...`
+* Edits an existing student's information found in Tracey. The student is referred by an index, displayed alongside him in the student list.
+* To edit a student, you can use the command `edit` on the student's `INDEX` with the optional fields of information to be changed.
+  The index **must be a positive integer** 1, 2, 3, …
+* You have to provide at least one of the optional fields of information.
```

Comment:
What does this line mean? which are the optional fields? if we are going to include this line, I think we should spell out clearly to the reader which are the optional and compulsory fields

Reply:
Thank you. I have made the description simpler.